pkg/api/v1alpha1/edgeconnect/edgeconnect_types.go

```diff
@@ -42,6 +46,10 @@ type EdgeConnectSpec struct { //nolint:revive
 	// Pull secret for your private registry
 	CustomPullSecret string `json:"customPullSecret,omitempty"`
 
+	// Adds custom root certificate from a configmap.
```

Comment:
```suggestion
	// Adds custom root certificate from a configmap. Put the certificate under certs within your configmap.
```

This is probably not decided yet, but it will probably work like that.